solr/core/src/test/org/apache/solr/update/RootFieldTest.java

```diff
@@ -124,4 +132,8 @@ public void testUpdateWithChildDocs() throws Exception {
     }
     client.commit();
   }
+
+  public EmbeddedSolrServer getSolrClient() {
+    return solrClientTestRule.getSolrClient();
+  }
```

Comment:
Isn't this already defined on EmbeddedSolrServerTestRule?  (if so, then we needn't specify this)